### ISSUE TITLE
Incorrect i18n on theme switcher

### ISSUE THREAD

**Issue by Almost-Senseless-Coder** [welpo/tabi#188]
# Bug Report

## Environment

Zola version: 0.17.2
tabi commit: Latest

## Expected Behavior

The `aria-label` should get internationalised correctly, showing up fully translated.

## Current Behavior

```
    {%- set aria_label = 'Toggle ' ~ dark_str ~ ' mode' -%}
```

This results in a partial translation. I'll see if I can fix this quick.

Also, setting the `aria-label` in the JS should no longer be necessary.

## Step to Reproduce

Switch on a screen reader of choice, select a language other than English and tab to the theme switcher. Result should be something like "Toggle <native word for dark> mode".
